File: clients/vscode/src/test/fixtures/csharp-small/Beta.cs
public class Beta {
    public int Compute(int a, int b) {
        var sum = a + b;
        var total = 0;
        for (var i = 0; i < 12; i++) {
            if (i % 2 == 0) {
                total += sum + i;
            } else {
                total += sum - i;
            }
        }
        if (total > 100) {
            total = total / 2;
        }
        return total * 2;
    }
}

File: crates/deslop/tests/fixtures/go-closure-signature-only/beta.go
package beta

// makeAccumulator carries the same closure signature as alpha.go and a
// deliberately different closure body: a running total over a counted
// loop. Nothing here is copy-pasted from alpha.go — only the parameter
// list and result types coincide, which is exactly the false positive
// #154 exists to kill.
func makeAccumulator(width int) func(name string, count int, active bool) (int, error) {
	return func(name string, count int, active bool) (int, error) {
		total := width
		for index := 0; index < count; index++ {
			total = total + index
		}
		return total, nil
	}
}
